docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+  amo-tools-suite:
+    build: .
+    container_name: amo-tools-suite-build
+    restart: always
+    command:
+      - /bin/sh
+      - -c
+      - |
+        cd /home/AMO-Tools-Suite/
+        npm run test-wasm-stop
+        npm run test-wasm
+    volumes:
+      - ../AMO-Tools-Suite:/AMO-Tools-Suite
+    ports:
+      - "127.0.0.1:3000:3000"
```

Comment:
I wasn't able to access "127.0.0.1:3000" until I removed the localhost ip and just did `3000:3000`. My docker experience is quite low but it seems declaring the ip specifically makes it so the port is only listening on the container.

Reply:
this is not preferable, but since we are locally using to test only, yes I have made that change in my latest updates